protocol/messenger_mailserver_cycle.go

```diff
@@ -27,8 +27,9 @@ import (
 const defaultBackoff = 10 * time.Second
 const graylistBackoff = 3 * time.Minute
 const isAndroidEmulator = runtime.GOOS == "android" && runtime.GOARCH == "amd64"
+const isIosEmulator = runtime.GOOS == "ios" && runtime.GOARCH == "arm64"
```

Comment:
@jo-mut : can you check if this will work for folks running iOS Simulators on intel macbooks?
cc @shivekkhurana